gwpy/table/gravityspy.py

```diff
@@ -177,3 +189,44 @@ def _download_single_image(url):
             if isinstance(x, Exception):
                 x.args = ('Failed to read %s: %s' % (f, str(x)),)
                 raise x
+
+    @classmethod
+    def search(cls, uniqueID, howmany=10):
+        """If table contains Gravity Spy triggers `EventTable`
+
+        Parameters
+        ----------
+        uniqueID : `str`,
+            This is the unique 10 character hash that identifies
+            a Gravity Spy Image
+
+        howmany : `int`, optional, default: 10
+            number of similar images you would like
+
+        kwargs:
+
+        Returns
+        -------
+        Folder containing omega scans sorted by label
+        """
+        # back to pandas
+        try:
+            import pandas
+        except ImportError as exc:
+            exc.args = ('pandas is required to download triggers',)
+            raise
+
+        # Need to build the url call for the restful API
+        url = 'https://gravityspytools.ciera.northwestern.edu' + \
+            '/search/similarity_search_restful_API/?'
+        # The first varaible username for the url is unimportant
+        # and can permantly be 'sbc538'
+        username = 'sbc538'
+        url = url + 'username={0}&'.format(username)
+        url = url + 'howmany={0}&'.format(howmany)
+        # zooID can also be permanantly set to zero.
+        zooID = '0'
+        url = url + 'zooid={0}&'.format(zooID)
+        url = url + 'imageid={0}'.format(uniqueID)
```

Comment:
it might be cleaner to build the URL using something like

```python
        parts = {
            'username': username,
            'howmany': howmany,
            'zooid': 0,
            'imageid': uniqueID,
        }
        search = '&'.join('{}={}'.format(key, value) for
                          key, value in parts.items())
        url = '{}/?{}'.format(base, search)
```

which might make it easier to modify/add/remove URL query components.

Reply:
Excellent will make this change and the above shortly